src/go/vtunnel/README.md

```diff
@@ -40,17 +40,27 @@ You can simply run the e2e test:
  GOOS=windows go build
  GOOS=linux go build
 ```
+- Creat a configuration file, below is an example a config.yaml.
+ **Note** the configuration for both Peer and Host process is identical.
+ ```yaml
+ tunnel:
+  - handshakePort: 9090
+    vsockHostPort: 8989
+    peerAddress: 127.0.0.1
+    peerPort: 3030
+    upstreamServerAddress: 127.0.0.1:4444
+ ```
  - Move the vtunnel to the hyper-v VM and run the Peer process:
  ```bash
- ./vtunnel peer --handshake-port 9090 --vsock-port 8989 --listen-address 127.0.0.1:3030
+ ./vtunnel peer --configPath config.yaml
```

Comment:
Interesting; I'd have expected `--config-path` instead.

Reply:
@mook-as this was `--config-path` before, but you [suggested](https://github.com/rancher-sandbox/rancher-desktop/pull/2436#discussion_r902949910) changing it to `--configPath` 😹

Reply:
🤣 in that comment I meant to name the _variable_ (the one being assigned to) `configPath`, not the parameter; my bad for having a terrible review comment.

This isn't going to block merging this PR.